DEFI/ERC4626/README.md

```diff
@@ -0,0 +1,12 @@
+# ERC4626 - Full Spec
+
+A specification file of ERC4626.
+
+This spec can be checked against implementations of ERC4626.
+
+We have an implementation of an ERC4626 contract in `certora/harnesses/mixins/ERC4626AccountingHarness.sol`.
```

Comment:
say that the current implementation is solmate and give link to the original code (https://github.com/transmissions11/solmate) is it the latest? , maybe also rename src to solmate

Reply:
done.